codes/quantum/qubits/two_dimensional_hyperbolic_surface.yml

```diff
@@ -0,0 +1,106 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: two_dimensional_hyperbolic_surface
+physical: bits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Two-dimensional hyperbolic surface code'
+
+# First reference with the code (optional).
+introduced: '\cite{arXiv:1506.04029}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'Hyperbolic surface codes are closely related to the toric code (aka surface code) except the tessellations on the manifold are derived on a hyperbolic geometry (ie, non-Euclidean geometry, think saddle surfaces when defined on a 2D plane) instead of Eulicdean surfaces. For a tessellation involving regular polygons with \( r \) sides and \( s \) polygons meeting at each edge, the number of logical qubits (which sit at the center of each tile) is given by \( k = (1-2/r - 2/s) n + 2 \). Some possible tilings are {r,s}: {7,3}, {5,4}. For closed surfaces, two-dimensional hyperbolic surface codes have an asymptotically constant encoding rate $k/n$ with a distance scaling logarithmically $\log(n)$. Additionally, surfaces with more tiling (larger values of r, s) have higher encoding rates (more logical qubits per physical qubits) but stabilizers with larger weights. Given the stabilizers are measured on noisy circuits, lower weight stabilizers are preferable. Therefore, the family of hyperbolic codes generated from the {5,4} tiling may be more favorable since they have lower weight stabilizers. '
+
+protection: 'Protects against Pauli errors with distance \( d \propto \log(n) \). Approximate code parameters are \( [[n, (1-2/r - 2/s)  n + 2, \log(n) ]] \)'
```

Comment:
Why are these approximate?

Reply:
[Note to self: check this]

Reply:
It isn't approximate, my mistake. Deleted approximate